plugins/builds/get.js

```diff
@@ -23,27 +23,32 @@ module.exports = () => ({
             }
         },
         handler: (request, reply) => {
-            const factory = request.server.app.buildFactory;
+            const buildFactory = request.server.app.buildFactory;
+            const stepFactory = request.server.app.stepFactory;
 
-            return factory.get(request.params.id)
-                .then((model) => {
-                    if (!model) {
+            return Promise.all([buildFactory.get(request.params.id),
+                stepFactory.list({ params: { buildId: request.params.id } })])
+                .then(([buildModel, stepsModel]) => {
+                    if (!buildModel) {
                         throw boom.notFound('Build does not exist');
                     }
```

Comment:
Do we need to check `stepModel` as same as `buildModel` ?

Reply:
Thank you.
I fixed

Reply:
Do we need to add test cases for empty `stepsModel`?